definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "LiveFrom": "12/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AwaitingProcessServerService",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "CRU"
```

Comment:
Is there a reason why we need "CRU" on this one? :)

Reply:
The AC states "A Caseworker can trigger event 'serveByProcessServer' from 'GeneralConsiderationComplete'. This moves case into the state 'AwaitingProcessServerService'."
As we have two roles for Caseworker, i thought it was appropriate to have this